FirebaseAuth/Sources/SystemService/FIRAuthAPNSTokenManager.h

```diff
@@ -56,12 +60,21 @@ typedef void (^FIRAuthAPNSTokenCallback)(FIRAuthAPNSToken *_Nullable token,
  */
 - (instancetype)init NS_UNAVAILABLE;
 
-/** @fn initWithApplication:bundle
+#if TARGET_OS_WATCH
```

Comment:
Something like GDTCORPlatform to abstract `WKExtension` and `UIApplication` might be cleaner.

Reply:
Turns out these weren't actually needed - only for Phone Auth which was disabled already 😅 removed. Thanks for making me look a second time!